docs/changes/2.0.0.rst

```diff
@@ -80,15 +80,15 @@ Breaking Changes
   the environment). Instead, :class:`~.media.JSONHandler` can now be configured
   to use arbitrary ``dumps()`` and ``loads()`` functions. If you
   also need to customize :class:`~.HTTPError` serialization, you can do so via
-  :meth:`~.API.set_error_serializer`.
+  :meth:`~.App.set_error_serializer`.
```

Comment:
Let us not tamper with this file either. `2.0.0` is an already released version.

Reply:
I pushed a commit that should hopefully address this too.